cmd/createTeam.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright © 2023 Kondukto
+
+*/
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/kondukto-io/kdt/client"
+
+	"github.com/spf13/cobra"
+)
+
+// createTeamCmd represents the create project command
+var createTeamCmd = &cobra.Command{
+	Use:   "team",
+	Short: "creates a new team on Kondukto",
+	Run:   createTeamRootCommand,
+}
+
+func init() {
+	createCmd.AddCommand(createTeamCmd)
+
+	createTeamCmd.Flags().StringP("name", "n", "", "team name")
+	createTeamCmd.Flags().StringP("responsible", "r", "", "responsible user name")
+}
+
+func createTeamRootCommand(cmd *cobra.Command, _ []string) {
+	c, err := client.New()
+	if err != nil {
+		qwe(ExitCodeError, err, "could not initialize Kondukto client")
+	}
+
+	teamName, err := cmd.Flags().GetString("name")
+	if err != nil {
+		qwe(ExitCodeError, err, "failed to parse the name flag")
+	}
+
+	responsible, err := cmd.Flags().GetString("responsible")
+	if err != nil {
+		qwe(ExitCodeError, err, "failed to parse the responsible flag")
+	}
+
+	if teamName == "" {
+		qwm(ExitCodeError, "team name is required")
```

Comment:
I think this check should be done first

Reply:
You right, thank you 🍀 
https://github.com/kondukto-io/kdt/pull/84/commits/4952462218e6e0640e814086b99ca701a84eb934